lib/std/wordwrap.nim

```diff
@@ -0,0 +1,67 @@
+import unicode
+
+proc wordWrap*(s: string, maxLineWidth = 80,
```

Comment:
procs use verbs, so the name should be ``wrapWords``.

Reply:
That doesn't make sense. "wordwrap" can be a verb too. Using your argument, we'd then have "echoes", "aligns", "adds", etc.

Reply:
Another argument:

- This function is supposed to "echo"
- This function is supposed to "align"
- This function is supposed to "wordwrap" (sounds right!)

Reply:
Basically, use the infinitive verb without the "to": http://grammar.yourdictionary.com/parts-of-speech/verbs/infinitive-verb.html